service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyPersistenceIT.java

```diff
@@ -31,18 +36,15 @@
 import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
 import org.kaazing.k3po.junit.annotation.Specification;
 import org.kaazing.k3po.junit.rules.K3poRule;
-
-import java.net.URI;
-
-import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.junit.rules.RuleChain.outerRule;
+import org.kaazing.test.util.MethodExecutionTrace;
 
 public class HttpProxyPersistenceIT {
 
     @Rule
```

Comment:
Remove the Rule annotation as it is part of the chain. Do this for other places also in the changeset.